src/NzbDrone.Core/ImportLists/Simkl/User/SimklUserRequestGenerator.cs

```diff
@@ -21,7 +23,19 @@ public virtual ImportListPageableRequestChain GetListItems()
 
         private IEnumerable<ImportListRequest> GetSeriesRequest()
         {
-            var link = $"{Settings.BaseUrl.Trim()}/sync/all-items/shows/{((SimklUserListType)Settings.ListType).ToString().ToLowerInvariant()}";
+            var link = "";
+
+            switch (ShowType)
```

Comment:
Instead of being set on the generator this should come from settings, similar to `Settings.ListType`